osgar/drivers/kloubak.py

```diff
@@ -625,32 +625,32 @@ def update_drive_two_axles(self, timestamp, data):
 
         if self.process_packet(data):
-            stop = [0, 0, 0, 0]
-#
+#            stop = [0, 0, 0, 0]
+
             if self.verbose:
                 self.speed_debug_arr.append( [ timestamp.total_seconds(), self.v_fl, self.v_fr, self.v_rl, self.v_rr ] )
 
-            if self.desired_speed != 0: # and self.drive_mode == DriveMode.ALL: ?
-                if self.last_encoders_front_right is not None:
-                    v_fr_2 = int(round(self.v_fr / (VESC_REPORT_FREQ * SPEED_ENC_SCALE)))
-                    self.publish('can', CAN_triplet(0x31, list(struct.pack('>i', v_fr_2))))  # front right
-                if self.last_encoders_front_left is not None:
-                    v_fl_2 = int(round(self.v_fl / (VESC_REPORT_FREQ * SPEED_ENC_SCALE)))
-                    self.publish('can', CAN_triplet(0x32, list(struct.pack('>i', v_fl_2))))  # front left
-                if self.last_encoders_rear_right is not None:
-                    v_rr_2 = int(round(self.v_rr / (VESC_REPORT_FREQ * SPEED_ENC_SCALE)))
-                    self.publish('can', CAN_triplet(0x33, list(struct.pack('>i', v_rr_2))))  # rear right
-                if self.last_encoders_rear_left is not None:
-                    v_rl_2 = int(round(self.v_rl / (VESC_REPORT_FREQ * SPEED_ENC_SCALE)))
-                    self.publish('can', CAN_triplet(0x34, list(struct.pack('>i', v_rl_2))))  # rear left
+            #if self.desired_speed != 0: # and self.drive_mode == DriveMode.ALL: ?
+            if self.last_encoders_front_right is not None:
+                v_fr_2 = int(round(self.v_fr / (VESC_REPORT_FREQ * SPEED_ENC_SCALE)))
+                self.publish('can', CAN_triplet(0x31, list(struct.pack('>i', v_fr_2))))  # front right
+            if self.last_encoders_front_left is not None:
+                v_fl_2 = int(round(self.v_fl / (VESC_REPORT_FREQ * SPEED_ENC_SCALE)))
+                self.publish('can', CAN_triplet(0x32, list(struct.pack('>i', v_fl_2))))  # front left
+            if self.last_encoders_rear_right is not None:
+                v_rr_2 = int(round(self.v_rr / (VESC_REPORT_FREQ * SPEED_ENC_SCALE)))
+                self.publish('can', CAN_triplet(0x33, list(struct.pack('>i', v_rr_2))))  # rear right
+            if self.last_encoders_rear_left is not None:
+                v_rl_2 = int(round(self.v_rl / (VESC_REPORT_FREQ * SPEED_ENC_SCALE)))
+                self.publish('can', CAN_triplet(0x34, list(struct.pack('>i', v_rl_2))))  # rear left
 
 #            elif self.drive_mode == DriveMode.FRONT ?
 
-            else:
-                self.publish('can', CAN_triplet(0x11, stop))  # right front
-                self.publish('can', CAN_triplet(0x12, stop))  # left front
-                self.publish('can', CAN_triplet(0x13, stop))  # right rear
-                self.publish('can', CAN_triplet(0x14, stop))  # left rear
+#            else:
```

Comment:
I wish this would cleaned up one day ... or as extra config param with default to brake during go(0)